alt_evaluator.py

```diff
@@ -19,6 +19,14 @@ def evaluate(self, model):
 
         with torch.no_grad():
             for _, (images, length_labels, digits_labels, paths) in enumerate(self._loader):
+            length_results = 0
+            digit1_results = 0
+            digit2_results = 0
+            digit3_results = 0
+            digit4_results = 0
+            digit5_results = 0
```

Comment:
What are the lines above for?

Reply:
There are for when there's an error in the lmdb file and the for loop doesn't run, it errors out without these

Reply:
Oh makes sense, is the `for _, (images, length_labels, digits_labels, paths) in enumerate(self._loader):` line above it extra then?

Reply:
oop that line shouldn't be there, will get rid of it